CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 ### Payments
 * [Fixed] STPPaymentHandler.handleNextAction allows payment methods that are delayed or require further customer action like like SEPA Debit or OXXO.
 
+### PaymentSheet
```

Comment:
Also can we update the docs, we have a callout mentioning ACH v2 is not supported.

Reply:
Hmm yeah I think we should mention the decoupled private beta in the CHANGELOG.  

Since opening this PR I learned we're blocked on https://jira.corp.stripe.com/browse/BANKCON-6731 - will update docs once ACHv2 is supported.